src/app/filecoin-plus/allocators/schemas/allocatorSchema.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from 'zod'
+
+const ApplicationSchema = z.object({
+  required_sps: z.string(),
+  required_replicas: z.string(),
+  allocation_bookkeeping: z.string(),
```

Comment:
```suggestion
  allocation_bookkeeping: z.string().nullable(),
```
Field can be null
https://github.com/filecoin-project/Allocator-Registry/blob/main/Allocators/1024.json